client/src/route/library/Library.tsx

```diff
@@ -1,27 +1,59 @@
 import * as React from 'react';
 import Layout from 'page/Layout';
 import TabComponent from 'components/tab/TabComponent';
-import Iframe from 'components/Iframe';
 import { TabData } from 'components/tab/subcomponents/TabRender';
-import { useURLforLIB } from 'util/envUtil';
-import { Typography } from '@mui/material';
+import { Paper, Typography } from '@mui/material';
+import AssetBoard from 'components/AssetBoard';
+import ShoppingCart from 'components/cart/ShoppingCart';
 import tabs from './LibraryTabData';
 
-function LibraryContent() {
-  const LIBurl = useURLforLIB();
-
+function useLibraryData() {
+  const assetBoardWithSuspense = (
+    pathToAssets: string,
+    privateRepo?: boolean
```

Comment:
I forgot to ask this basic question. How are the gitlab credentials passed onto the client/ code?

Reply:
As and when the authentication PR is merged, the corresponding headers get added here right?

Reply:
Exactly ;)